borrowing_app/views.py

```diff
@@ -36,7 +39,31 @@ def get_queryset(self):
     def get_serializer_class(self):
         if self.action in ("list", "retrieve"):
             return BorrowingListSerializer
+        elif self.action == "return_book":
+            return BorrowingReturnSerializer
         return self.serializer_class
 
     def perform_create(self, serializer):
         serializer.save(user=self.request.user)
+
+    @action(
+        detail=True,
+        methods=["get", "post"],
+        permission_classes=[IsAuthenticated],
+        url_path="return-book",
```

Comment:
![image](https://github.com/panicua/library-service/assets/37306964/a6000ee5-57f5-47e4-abff-a2b67f96c301)

Reply:
and we don't need GET method for this action if I got it right